Core/ModuleInstaller.cs

```diff
@@ -177,7 +177,15 @@ public void InstallList(ICollection<CkanModule> modules, RelationshipResolverOpt
                 }
             }
 
-            bool ok = User.RaiseYesNoDialog("\r\nContinue?");
+            bool ok;
+            if (User.GetType().ToString() == "CKAN.GUIUser")
```

Comment:
The old method that GUI used for suppressing this prompt was pretty bad, but this might be worse.

What about a new `bool confirmPrompt = true` parameter, which only GUI would pass as false?

Reply:
Yeah, I don't like my solution either, but I searched for a way to get around this without adding something to the interface and all the User classes, because it is only needed for the GUI. But maybe I should do it like this.